### PR TITLE
Added power state to Hana Instance

### PR DESCRIPTION
**Pull request by PakDLiu**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/preview/2017-11-03-preview/hanaonazure.json [hunk 1]
```diff
@@ -220,6 +220,11 @@
                     "type": "string",
                     "description": "Resource location"
                 },
+                "powerState": {
```
Comment (dsgouda) [inline specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/preview/2017-11-03-preview/hanaonazure.json]: This cannot be a top level resource property. Must be a part of the `properties` propety

Comment (PakDLiu) [reply specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/preview/2017-11-03-preview/hanaonazure.json]: Got it, updated

Comment (dsgouda) [reply specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/preview/2017-11-03-preview/hanaonazure.json]: Please check if this is how the service is consuming/returning the resource information

Comment (PakDLiu) [reply specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/preview/2017-11-03-preview/hanaonazure.json]: I've just updated our API to move the power state. here is a sample:
```
{
    "id": "/subscriptions/c5c14913-17c5-4bb7-a8d4-5fb40d0a4823/resourceGroups/rg10004/providers/Microsoft.HanaOnAzure/hanaInstances/westusbmiName",
    "name": "westusbmiName",
    "type": "Microsoft.HanaOnAzure/hanaInstances",
    "location": "westus",
    "tags": {
     "owner": "alias4"
    },
    "properties": {
     "hanaInstanceId": "2e30e358-7ceb-42d0-b7f4-cb7e41e1a8bc",
     "powerState": "started",
     "hardwareProfile": {
      "hardwareType": "Cisco_UCS",
      "hanaInstanceSize": "S72"
     },
     "networkProfile": {
      "networkInterfaces": [
       {
        "ipAddress": "123.123.123.123"
       }
      ],
      "circuitId": "/subscriptions/c5c14913-17c5-4bb7-a8d4-5fb40d0a4823/resourceGroups/rg10004/providers/Microsoft.Network/expressRouteCircuits/westusbmiName"
     },
     "storageProfile": {
      "nfsIpAddress": "123.123.123.124"
     },
     "osProfile": {
      "computerName": "westusbmiName",
      "osType": "SLES 12 SP2",
      "version": "12 SP2"
     }
    }
   }
```

Comment (dsgouda) [reply specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/preview/2017-11-03-preview/hanaonazure.json]: Looks good.
As a sidenote, it looks like `powerstate` could be an enum, you could simply use an `x-ms-enum`

Comment (PakDLiu) [reply specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/preview/2017-11-03-preview/hanaonazure.json]: `powerstate` is an enum now

**Review by dsgouda — APPROVED** [review]

**Comment by dsgouda** [comment]
LGTM

**Pull request merged**